hyperspy/tests/io/test_hdf5.py

```diff
@@ -229,3 +230,33 @@ def test_rgba16():
         "npy_files",
         "test_rgba16.npy"))
     nt.assert_true((s.data == data).all())
+
+
+class TestLoadingOOMReadOnly:
+
+    def setUp(self):
+        s = Signal(np.empty((5, 5, 5)))
+        s.save('tmp.hdf5')
+        self.shape = (10000, 10000, 100)
```

Comment:
It does not - it just creates an empty array that would take much more space than commonly available. Since there is no real data there, the actual file size is ~19KB